image/spec/v1.3.md

```diff
@@ -0,0 +1,21 @@
+# Docker Image Specification v1.3.0
```

Comment:
I would suggest not to create new "versioned" files in this directory. Once we migrate the spec to a separate module, I'm planning to remove the versioned files from the new repository.